generated/Pet.h

```diff
@@ -0,0 +1,115 @@
+#ifndef ORG_OPENAPITOOLS_MODEL_Pet_H_
+#define ORG_OPENAPITOOLS_MODEL_Pet_H_
+
+#include "serialization.h"
+
+#include <optional>
+
+#include "Tag.h"
+#include <string>
+#include "Category.h"
+#include <vector>
+
+namespace org {
+namespace openapitools {
+namespace model {
+
+/**
+ * \brief A pet for sale in the pet store
+ */
+struct Pet {
+    static const char *dtoName() { return "swagger::dto::Pet"; }
+
+    // enums
+    /**
+     * \brief pet status in the store
+     */
+    enum StatusEnum {
```

Comment:
You might want to define those as `enum class`. Otherwise, if a model as two enums with the same enum values, [we'll run into compilation errors](https://godbolt.org/z/dO8hlE).